tests/python/topi/python/test_topi_conv2d_nhwc_winograd.py

```diff
@@ -114,6 +111,8 @@ def check_device(device):
     check_device(devices)
 
 
+@requires_cuda
```

Comment:
requires_cuda imply requires gpu?

Reply:
perhaps it is better to use the abs name: `tvm.testing.requires_cuda` to give better context, do the same thing for other annotations